klippy/extras/display/uc1701.py

```diff
@@ -13,6 +13,7 @@
 TextGlyphs = { 'right_arrow': '\x1a', 'degrees': '\xf8' }
 
 class DisplayBase:
+    x_offset = 0
     def __init__(self, io, columns=128):
```

Comment:
It would be preferable if `x_offset` was passed as a parameter to the constructor: `def __init__(self, io, columns=128, x_offset=0):`